Components/Alerts/alerts.html

```diff
@@ -0,0 +1,167 @@
+                    style="width: 1024px; height: 1082px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Don't use inline styling

Reply:
Hey, This iframe style is injected by carbon highlight. I have not added inline style.